sdks/go/pkg/beam/core/runtime/graphx/coder.go

```diff
@@ -39,6 +39,7 @@ const (
 
 	urnGlobalWindow   = "beam:coder:global_window:v1"
 	urnIntervalWindow = "beam:coder:interval_window:v1"
+	urnSessionWindow  = "beam:coder:session_window:v1"
```

Comment:
If you want to use a well known name which isn't scoped to Go, you'll want to add it to the set here https://github.com/apache/beam/blob/e0cbd2aa7e371c75511544ab78075d54f3f086ca/model/pipeline/src/main/proto/beam_runner_api.proto#L616

The other well known window coders don't do a good job of it but you should really specify the encoding, look at beam:coder:iterable:v1 or beam:coder:timer:v1 for examples.